src/DI/TranslationExtension.php

```diff
@@ -392,10 +391,10 @@ protected function beforeCompileLogging(array $config)
 		if ($config['logging'] === TRUE) {
 			$translator->addSetup('injectPsrLogger');
 
-		} elseif (is_string($config['logging'])) { // channel for kdyby/monolog
-			$translator->addSetup('injectPsrLogger', [
-				new Statement(sprintf('@%s::channel', KdybyLogger::class), [$config['logging']]),
-			]);
+		//} elseif (is_string($config['logging'])) { // channel for kdyby/monolog
```

Comment:
I'd just throw this `elseif` branch away if we're not going to use it.

Reply:
OK, deleted.